src/components/skiplink-list/skiplink-list.ts

```diff
@@ -70,19 +59,9 @@ export class SbbSkiplinkListElement extends SlotChildObserver(LitElement) {
     this._links = links;
   }
 
-  public override connectedCallback(): void {
-    super.connectedCallback();
-    this._handlerRepository.connect();
-  }
-
-  public override disconnectedCallback(): void {
-    super.disconnectedCallback();
-    this._handlerRepository.disconnect();
-  }
-
   protected override render(): TemplateResult {
     let ariaLabelledByAttribute: Record<string, string> = {};
-    if (this._namedSlots.title || this.titleContent) {
+    if (this._namedSlots.slots.has('title') || this.titleContent) {
```

Comment:
As commented elsewhere, maybe condition is not needed at all as id is always there now. If still needed, it should update when condition changes (requestupdate or callback)